de.tudarmstadt.ukp.wikipedia.revisionmachine/src/main/java/de/tudarmstadt/ukp/wikipedia/revisionmachine/common/logging/messages/consumer/SQLConsumerLogMessages.java

```diff
@@ -85,17 +82,17 @@ public static void logReadTaskOutOfMemoryError(final Logger logger,
 	{
 
 		if (task != null) {
-			logger.logError(Level.WARNING, "Error while reading a task: "
+			logger.logError(Level.WARN, "Error while reading a task: "
```

Comment:
Shouldn't these calls be using placeholders as well instead of doing string concatenation?

Reply:
For this case, the class `de.tudarmstadt.ukp.wikipedia.revisionmachine.common.logging.Logger` does not handle placeholders/parameters at all. Therefore, no chance here to beautify the code. If we had placeholder replacement, then - obviously - it would be a reasonable change.